mux.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net"
 	"time"
 
-	muxv3 "go.sia.tech/mux/v3"
+	muxv3 "go.sia.tech/mux/v2/v3"
```

Comment:
definitely one of the import paths of all time

maybe we should lift v3 into the top-level, and push the version to v3?